### DISCUSSION TITLE
Projects (beta): Table cells with dropdowns should immediately open when you click them

### DISCUSSION THREAD

**Discussion by brettgoulder** [community/community#4728] [Projects and Issues] [answered]
**Current behavior**
The current behavior is you click a cell that has a dropdown, such as Status, it focuses on the cell and then you need to click once more to open the dropdown. This is especially annoying for entering tasks.

**Expected behavior**
If you click a table cell that has a dropdown, I expect the dropdown to immediately open.

**Comment by azenMatt** [comment, marked as answer]
One way to do this currently is to hit the arrow on the right hand side of the drop down, that will open the selection immediately

**Comment by Alkarar1122** [comment]
One way to do this currently is to hit the arrow on the right hand side of the drop down, that will open the selection immediately

**Comment by CasperSocio** [comment]
It would be great if the dropdown opened when clicking the cell.
